qiskit/aqua/algorithms/classifiers/vqc.py

```diff
@@ -211,6 +212,10 @@ def _build_parameterized_circuits():
             feat_map_support = isinstance(self._feature_map, QuantumCircuit) \
                 or self._feature_map.support_parameterized_circuit
 
+            # cannot transpile the RawFeatureVector
+            if isinstance(self._feature_map, RawFeatureVector):
+                feat_map_support = False
+
```

Comment:
Maybe at some point once the feature map type is gone this logic/comments could be cleaned up. Without the comment about not be able to transpile, despite presumably it being parameterized it would be hard to tell from the variable names what they imply nowadays. And presumably though it instance checks for QuantumCircuit that need not have parameters in general, though of course our library ccts do.

Reply:
Yes it should. When we remove the feature map and varform types we can rework this 🙂 Though we'd still need a check to not transpile the `RawFeatureVector` as long as it is not fully bound